features/conference/components/Conference.js

```diff
@@ -18,13 +27,25 @@ class Conference extends Component {
      */
     render() {
         return (
-            <ConferenceContainer>
+            <ConferenceContainer onPress = { this.onPress.bind(this) }>
                 <LargeVideo/>
-                <Toolbar navigator = { this.props.navigator }/>
-                <FilmStrip/>
+                <Toolbar
```

Comment:
I wasn't talking about the toolbarVisible, but rather all toolbar buttons and their states. In the future we'll have more and more components in there, but even now do we keep the state of the flipCamera, so that we can initialise it properly if we have already flipped it and then removed the toolbar from the DOM. Do you see what I mean? For the filmStrip we have all the internal states of each thumbnail. So I wasn't sure that we take care of all those things and didn't want to risk it. I'm not even sure that we'd like to do it that way, but for a start it sounded risky.